content/en/docs/tasks/debug-application-cluster/debug-service.md

```diff
@@ -23,11 +23,24 @@ sees.  The simplest way to do this is to run an interactive busybox Pod:
 ```none
 kubectl run -it --rm --restart=Never busybox --image=gcr.io/google-containers/busybox sh
 ```
-
 {{< note >}}
 If you don't see a command prompt, try pressing enter.
 {{< /note >}}
 
+Alternatively you can run the command directly from the pod. For example, let us say you wish to run `nslookup `for an existing Service named `servicesample`, and will store the output in `nslookupout.txt`. A command that achieves this is:
+
+```none
```

Comment:
nit:
this is valid shell; we could highlight it as such

Reply:
```suggestion
```sh
```